docs/examples/route53/.gitignore

```diff
@@ -1,2 +1,3 @@
-.terraform
+.terraform*
+terraform.tfstate*
```

Comment:
inspired by [Terraform.gitignore](https://github.com/github/gitignore/blob/master/Terraform.gitignore);  consider sensitive `*.tfvars`

Reply:
@kuritka we are not passing tfvars here at all, but doing it through the module invocation see `main.tf`

Reply:
@ytsarev , ah, clear now. Thx